alpha/apps/kaltura/modules/extwidget/actions/serveFlavorAction.class.php

```diff
@@ -366,6 +368,7 @@ public function execute()
 		$captionLanguages = $this->getRequestParameter('captions', '');
 		$this->pathOnly = $this->getRequestParameter('pathOnly', false);
 		self::$preferredStorageId = $this->getRequestParameter('preferredStorageId');
+		self::$fallbackStorageId = $this->getRequestParameter('fallbackStorageId');
```

Comment:
Must add default if param not exist here

Reply:
getRequestParameter already have a default value of null in case the parm is not provided in the request